### ISSUE TITLE
Incorrect column type for reviews

### ISSUE THREAD

**Issue by LnLcFlx** [IgnisDa/ryot#413]
I migrated following all [instructions](https://ignisda.github.io/ryot/migration.html#from-v2-to-v3) for the third time now, but I get the following errors when starting the new release.
Also, no content is being displayed in ryot. 

```
ryot  | 2023-10-20T12:10:09.955069Z  INFO ryot: Running version: 3.0.0
ryot  | 2023-10-20T12:10:09.979552Z  INFO ryot: Using database backend: PostgreSQL
ryot  | 2023-10-20T12:10:09.980534Z  INFO sea_orm_migration::migrator: Applying all pending migrations
ryot  | 2023-10-20T12:10:09.982540Z  INFO sea_orm_migration::migrator: No pending migrations
ryot  | 2023-10-20T12:10:09.984841Z  INFO ryot: Using timezone: Etc/GMT
ryot  | 2023-10-20T12:10:09.989983Z  INFO ryot: Listening on: [::]:8000
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:331:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:318:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:331:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:318:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:318:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:331:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:318:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:318:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:318:10:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/miscellaneous/resolver.rs:4302:14:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/miscellaneous/resolver.rs:4302:14:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 501) }))
```

**Comment by IgnisDa** [comment]
Ah yes this seems to be a big i introduced. Sorry for that. Can you send me the database backup you made so that I can push a fixed docker image?

**Comment by IgnisDa** [comment]
I think you have the same problem as https://github.com/IgnisDa/ryot/issues/411.

**Comment by LnLcFlx** [comment]
I did follow step 1&2 though.

**Comment by IgnisDa** [comment]
DId you regenerate the summary? If yes, please send me the export you made of the database to my email (in my GH profile).

**Comment by IgnisDa** [comment]
From our email conversation.

> Hey I imported the dump into my instance. It seems you did not regenerate the summary.
> 
> - Import the above dump into your instance
> - Regenerate the summary and change any preference
> - Then follow the migration steps as usual (replacing v2.24.1 with v2.24.2 which will be published in around 30 minutes)

Please LMK here if this still does not fix your issue.

**Comment by LnLcFlx** [comment]
I followed the steps as before with my old database but replacing v2.24.1 with v2.24.2 and the same issue arises.

**Comment by IgnisDa** [comment]
Did you wait for summary calculation to complete?

**Comment by LnLcFlx** [comment]
That's it! Working now! So sorry...

**Comment by LnLcFlx** [comment]
Wait I still think the migration guide is wrong.
It only worked for me after I repeated step 1 on v2.24.2 aswell.

**Comment by LnLcFlx** [comment]
And I just now found out that reviewing content does not work

```
ryot  | 2023-10-20T17:29:59.694039Z  INFO ryot: Running version: 3.0.6
ryot  | 2023-10-20T17:29:59.713196Z  INFO ryot: Using database backend: PostgreSQL
ryot  | 2023-10-20T17:29:59.713757Z  INFO sea_orm_migration::migrator: Applying all pending migrations
ryot  | 2023-10-20T17:29:59.715497Z  INFO sea_orm_migration::migrator: No pending migrations
ryot  | 2023-10-20T17:29:59.717997Z  INFO ryot: Using timezone: Etc/GMT
ryot  | 2023-10-20T17:29:59.723211Z  INFO ryot: Listening on: [::]:8000
ryot  | thread 'tokio-runtime-worker' panicked at apps/backend/src/miscellaneous/resolver.rs:3854:54:
ryot  | called `Result::unwrap()` on an `Err` value: Query(SqlxError(Database(PgDatabaseError { severity: Error, code: "42804", message: "column \"comments\" is of type json but expression is of type jsonb[]", detail: None, hint: Some("You will need to rewrite or cast the expression."), position: Some(Original(145)), where: None, schema: None, table: None, column: None, data_type: None, constraint: None, file: Some("parse_target.c"), line: Some(587), routine: Some("transformAssignedExpr") })))
ryot  | note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
```

If that bug is unrelated I will open a new issue.

**Comment by IgnisDa** [comment]
So i take it that the upgrade worked? I'll look into reviews.

**Comment by SirMartin** [comment]
I had also an issue following the steps. I got this. I will try as @LnLcFlx said, I will try at repeat the Clean and regenerate after updating to `2.24.2`

```
thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:305:10:
called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 496) }))
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:292:10:
called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 496) }))
thread 'tokio-runtime-worker' panicked at apps/backend/src/utils.rs:292:10:
called `Result::unwrap()` on an `Err` value: Query(SqlxError(ColumnDecode { index: "\"summary\"", source: Error("missing field `media_interacted_with`", line: 1, column: 496) }))
```

**Comment by IgnisDa** [comment]
Alright LMK if you have any problems.

**Comment by IgnisDa** [comment]
@LnLcFlx Please try `v3.0.7` when it is released. It should fix the reviews problem. Thanks for reporting.

**Comment by SirMartin** [comment]
@IgnisDa Probably you should include in the migration guide after step 4, maybe as a 4a step, to do again the "Clean and regenerate". I made it before change from 2.23.2 to 2.24.2, and now I did it again before move from 2.24.2 to 3.0.6 and worked like a charm.
